### PR TITLE
fix(docs): fix the docs for this plugin

### PR DESCRIPTION
**Pull request by shrutimantri**

### CODE REVIEW

File: src/main/java/io/kestra/plugin/spark/AbstractSubmit.java [hunk 1]
```diff
@@ -41,54 +41,54 @@ public abstract class AbstractSubmit extends Task implements RunnableTask<Script
...
-        title = "command line arguments for the application."
+        title = "Command line arguments for the application."
     )
     @PluginProperty(dynamic = true)
     private List<String> args;
 
     @Schema(
         title = "Adds a file to be submitted with the application.",
-        description = "Must be Kestra internal storage url"
+        description = "Must be Kestra internal storage URI."
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/spark/AbstractSubmit.java:65]:
```suggestion
        description = "Must be an internal storage URI."
```

File: src/main/java/io/kestra/plugin/spark/JarSubmit.java [hunk 2]
```diff
@@ -20,7 +20,7 @@
 @Getter
 @NoArgsConstructor
 @Schema(
-    title = "Submit a jar spark job to remote cluster"
+    title = "Submit a Spark job using JAR to remote cluster."
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/spark/JarSubmit.java:23]:
```suggestion
    title = "Submit a Spark job to a remote cluster using a JAR file."
```

File: src/main/java/io/kestra/plugin/spark/JarSubmit.java [hunk 3]
```diff
@@ -36,24 +36,24 @@
 )
 public class JarSubmit extends AbstractSubmit {
     @Schema(
-        title = "the main application resource",
-        description = "This should be the location of a jar file for Scala/Java applications, or a python script for PySpark applications.\n" +
-            "Must be Kestra internal storage url"
+        title = "The main application resource.",
+        description = "This should be the location of a JAR file for Scala/Java applications, or a Python script for PySpark applications.\n" +
+            "Must be Kestra internal storage URI."
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/spark/JarSubmit.java:41]:
```suggestion
            "Must be an internal storage URI."
```

File: src/main/java/io/kestra/plugin/spark/PythonSubmit.java [hunk 4]
```diff
@@ -26,7 +26,7 @@
 @Getter
 @NoArgsConstructor
 @Schema(
-    title = "Submit a pyspark job to remote cluster"
+    title = "Submit a PySpark job to remote cluster."
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/spark/PythonSubmit.java:29]:
```suggestion
    title = "Submit a PySpark job to a remote cluster."
```

File: src/main/java/io/kestra/plugin/spark/PythonSubmit.java [hunk 5]
```diff
@@ -70,15 +70,15 @@
...
     )
     @PluginProperty(dynamic = true)
     @NotNull
     private String mainScript;
 
     @Schema(
-        title = "Adds a python file / zip / egg to be submitted with the application.",
-        description = "Must be Kestra internal storage url"
+        title = "Adds a Python file / zip / egg to be submitted with the application.",
+        description = "Must be Kestra internal storage URI."
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/spark/PythonSubmit.java:81]:
```suggestion
        title = "Adds a Python file/zip/egg package to be submitted with the application.",
        description = "Must be an internal storage URI."
```

File: src/main/java/io/kestra/plugin/spark/RSubmit.java [hunk 6]
```diff
@@ -23,7 +23,7 @@
 @Getter
 @NoArgsConstructor
 @Schema(
-    title = "Submit a R job to remote cluster"
+    title = "Submit a R job to remote cluster."
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/spark/RSubmit.java:26]:
```suggestion
    title = "Submit an R job to a remote cluster."
```

File: src/main/java/io/kestra/plugin/spark/SparkCLI.java [hunk 7]
```diff
@@ -27,12 +27,12 @@
...
 @NoArgsConstructor
 @Schema(
-    title = "Execute spark CLI commands."
+    title = "Execute Spark CLI commands."
 )
 @Plugin(
     examples = {
         @Example(
-            title = "Submit a Python Spark Job to a master",
+            title = "Submit a PySpark job to a master.",
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/spark/SparkCLI.java:35]:
```suggestion
            title = "Submit a PySpark job to a master node.",
```

File: src/main/java/io/kestra/plugin/spark/package-info.java [hunk 8]
```diff
@@ -1,6 +1,6 @@
 @PluginSubGroup(
     description = "This sub-group of plugins contains tasks for submitting Apache Spark jobs.\n" +
-        "Apache Spark is a multi-language engine for executing data engineering, data science, and machine learning on single-node machines or clusters.",
+        "Apache Spark is a multi-language engine for executing data engineering, data science, and machine learning processing tasks on single-node machines or clusters.",
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/spark/package-info.java:3]:
```suggestion
        "Apache Spark is a multi-language engine for executing data engineering, data science, and machine learning jobs on distributed computing clusters.",
```

**Review by anna-geller — CHANGES_REQUESTED** [review]

File: src/main/java/io/kestra/plugin/spark/RSubmit.java (hunk 6 above)
Comment (shrutimantri) [reply src/main/java/io/kestra/plugin/spark/RSubmit.java:26]: Changed it to "Submit a R job to a remote cluster."

File: src/main/java/io/kestra/plugin/spark/RSubmit.java [hunk 9]
```diff
@@ -23,7 +23,7 @@
 @Getter
 @NoArgsConstructor
 @Schema(
-    title = "Submit a R job to remote cluster"
+    title = "Submit a R job to a remote cluster."
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/spark/RSubmit.java:26]:
```suggestion
    title = "Submit an R job to a remote cluster."
```

**Review by anna-geller — APPROVED** [review]

**Pull request merged**